app/helpers/fact_values_helper.rb

```diff
@@ -66,4 +66,24 @@ def fact_breadcrumbs
       switchable: true
     )
   end
+
+  def is_escaped_value(value)
+    value != CGI.escapeHTML(value)
+  end
+
+  def escaped_warning_title
+    _("contains special characters")
+  end
+
+  def escaped_warning_context
+    _("Search may fail or give you wrong results since it contains special characters that are query keywords such as < and >")
+  end
+
+  def escaped_warning_chart_context
```

Comment:
Looks like this and the following method aren't used anywhere?

Reply:
Thanks for noticing this. I will open a ticket and fix that as soon as possible.

Thanks again and sorry for this mess.